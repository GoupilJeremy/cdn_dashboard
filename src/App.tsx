import React, { useState } from 'react';
import Login from './components/login/LoginForm';

const App: React.FC = () => {

  const [isLogged, setIsLogged] = useState(false);

  const toggleLogin = (isLogged: boolean) => {
    setIsLogged(isLogged);
  }

  return (
    <div className="App">
      {!isLogged && <Login handleLogin={toggleLogin} />}
    </div>
  );
}

export default App;
