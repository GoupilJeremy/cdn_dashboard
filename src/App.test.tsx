import { render } from '@testing-library/react';
import App from './App';

test('renders the App', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
