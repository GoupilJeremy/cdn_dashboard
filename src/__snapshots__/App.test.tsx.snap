// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the App 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <form>
      <label
        for="identifiant"
      >
        Identifiant
      </label>
      <input
        id="identifiant"
        name="identifiant"
        type="text"
        value=""
      />
      <br />
      <label
        for="password"
      >
        Password
      </label>
      <input
        id="password"
        name="password"
        type="password"
        value=""
      />
      <input
        type="submit"
        value="Submit"
      />
    </form>
  </div>
</DocumentFragment>
`;
